### PR TITLE
Controllers created for blog-model.

### PR DESCRIPTION
**Pull request by samcrudge**
https://github.com/vlucas/valitron

validator package im using

### CODE REVIEW

File: src/Controllers/createNewEntryController.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+namespace App\Controllers;
+
+use App\Abstracts\Controller;
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+
+class createNewEntryController extends Controller
```
Comment (turnerror) [inline src/Controllers/createNewEntryController.php]: classes and files should be pascal case.

File: src/Db/dbConnection.php [hunk 2]
```diff
@@ -4,10 +4,10 @@
 
 use PDO;
 
-class DbConnection
+class dbConnection
```
Comment (turnerror) [inline src/Db/dbConnection.php]: not sure why you changed this from pascal case

File: src/Controllers/createNewEntryController.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        $this->blogModel = $blogModel;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args)
+    {
+        $addNewEntry = $request->getParsedBody();
+        $result = $addNewEntry['newBlogEntry'];
+        if ($result == true)
+        {
```
Comment (turnerror) [inline src/Controllers/createNewEntryController.php]: what does your json structure look like. I don't really get this check, also always use ===

File: src/Controllers/createNewEntryController.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            $dbResult = $this->blogModel->createNewEntry($addNewEntry);
```
Comment (turnerror) [inline src/Controllers/createNewEntryController.php]: not using `$dbResult` anywhere here

File: src/Controllers/deleteEntryController.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    public function __construct($blogModel)
+    {
+        $this->blogModel = $blogModel;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args)
+    {
+        $deletedEntry = $request->getParsedBody();
+        $dbResult = $this->blogModel->deleteEntry($deletedEntry);
+        return $response->withHeader('Location', '/');
```
Comment (turnerror) [inline src/Controllers/deleteEntryController.php]: same as above, if the deleted entry fails, you have no way of handling it.

**Review by turnerror — CHANGES_REQUESTED** [review]
left some comments, a lot of them like pascal case for class names and files apply across the board so have fun

File: src/Controllers/createNewEntryController.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        }
+        return $response->withHeader('location', '/');
+    }
+
+}
```
Comment (turnerror) [inline src/Controllers/createNewEntryController.php]: also missing new line at end of file.

File: src/Controllers/createNewEntryController.php (hunk 1 above)
Comment (samcrudge) [reply src/Controllers/createNewEntryController.php]: Fixed.

File: src/Db/dbConnection.php (hunk 2 above)
Comment (samcrudge) [reply src/Db/dbConnection.php]: Fixed.

File: src/Controllers/createNewEntryController.php (hunk 3 above)
Comment (samcrudge) [reply src/Controllers/createNewEntryController.php]: Good catch i missed that, was meant to be ===

**Pull request merged**